Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
     "iota-core",
+    # "bindings/python/native",
```

Comment:
Maybe we should create an extra workspace in the cargo.toml in the python folder like we have for the node binding now?